app/assets/javascripts/admin/addon/components/settings-editor.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="settings-editor">
+  <AceEditor
+    @mode={{"html"}}
```

Comment:
I believe the syntax here needs to be 
```suggestion
    @mode="html"
```

without the curlies, but also, this should be "json" vs "html", so we get JSON highlighting. But I'm just noticing that we don't load the json mode for ACE. Our wiring of ACE is a bit custom, if you're curious, you could take a look at `javascript.rake` and see if you can wire it. (It's a nice-to-have.)